google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/MessageDispatcher.java

```diff
@@ -264,6 +258,25 @@ void sendAckOperations(
     this.clock = clock;
   }
 
+  public void start() {
+    pendingAcksAlarm =
+        systemExecutor.scheduleWithFixedDelay(
+            new Runnable() {
+              @Override
+              public void run() {
+                try {
+                  processOutstandingAckOperations();
+                } catch (Exception e) {
```

Comment:
Is Throwable better than Exception for catching everything?

Reply:
Done